scene/gui/color_picker.cpp

```diff
@@ -624,26 +624,34 @@ void ColorPicker::_screen_pick_pressed() {
 }
 
 void ColorPicker::_focus_enter() {
+	bool has_ctext_focus = false;
```

Comment:
This can be simplified a bit with:
```
bool has_ctext_focus = c_text->has_focus();
if (has_ctext_focus) {
	c_text->select_all();
} else {
...
```